.ci/scripts/test_model.sh

```diff
@@ -152,32 +152,34 @@ test_model_with_qnn() {
 
   export LD_LIBRARY_PATH=$QNN_SDK_ROOT/lib/x86_64-linux-clang/
   export PYTHONPATH=$EXECUTORCH_ROOT/..
+  # QNN support fp16 only
```

Comment:
I think listed examples are compiled with 8bit fixed precision. Only `mobilenet_v3` in fp16 now and we'll change it to quantized version soon.

Reply:
@haowhsu-quic Maybe the info I got from Chen is outdated. Thanks for the clarification. BTW, since there is only one dtype for each model, isn't better to update the hard-coded name for each model to include the dtype info?

Reply:
Yes, will change those files in another PR. Models are expected to be quantized in general, only few of them encountering accuracy issues will stay in fp16 temporally.
Thank you for helping this.